ethcore/src/miner/miner.rs

```diff
@@ -189,14 +190,20 @@ pub struct Miner {
 impl Miner {
 	/// Creates new instance of miner without accounts, but with given spec.
 	pub fn with_spec(spec: &Spec) -> Miner {
+		let author = Address::default();
+		let is_sealer = spec.engine.is_sealer(&author);
```

Comment:
Perhaps a default implementation of a trait could do this.

For me it would be a bit clearer if the code looked like:
`let can_seal = spec.engine.can_seal();`
instead of checking if default address is able to seal - esp. it's already used in at least two places.